generated/nidmm/session.py

```diff
@@ -186,6 +196,7 @@ def __init__(self, resource_name, id_query=0, reset_device=False, options_string
         self.library = library.get_library()
         self.vi = 0  # This must be set before calling _init_with_options.
         self.vi = self._init_with_options(resource_name, id_query, reset_device, options_string)
+        self.acquisition = Acquisition(self)
```

Comment:
Methods aren't context managers. As far as I have been able to determine, it has to be a class with __enter__ and __exit__ methods or a function (not a method) with a \@contextmanager decoration.

Inner classes do not have access to instance variables like in other languages. This is why I create an object and give it a session object to hold on to. Otherwise it was looking like `session.Acquisition(session)` which was redundant. As I was looking around, either not using inner classes at [all](https://stackoverflow.com/questions/2024566/access-outer-class-from-inner-class-in-python) or using a [property](https://stackoverflow.com/questions/36601006/python-accessing-outer-private-functions-from-inner-class) was the answer: